CfCT/manifest.yaml

```diff
@@ -0,0 +1,36 @@
+---
+region: us-east-1
+version: 2021-03-15
+resources:
+    # NewRelic Hub stackset must be included in the manifest above the NewRelic spoke stackset
+  - name: newrelic-hub
+    resource_file: templates/newrelic-hub-cfct.yaml
+    parameter_file: parameters/newrelic-hub-cfct.json
```

Comment:
I don't think we need separate parameter file. In CTCF v2, parameters are part of manifest file. Isn't it?

Reply:
I can modify this to keep single manifest file.

For future reference, Is that the preference for CfCT? I don't see any specific guidance on the developer guide to avoid parameter file.